config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [
+  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
```

Comment:
is this a typo here should it not be `:password` rather than `:passw`? :)

Reply:
thank you for double-checking this.  actually tbh not sure about it, this file was generated automatically when I run` rails app:update`, depending how the `filter_parameters`, as the` :password` is included in `:passw` in a way, let me find it out more

Reply:
@willmcb,  couldn't find any place whether :passw includes key word of  'password' or not although I believe it should.  So I decided to add :password into the above list as well which make sure it did filter password key word out of log

Reply:
@willmcb , I tested this PR on QA, it looks fine, could you approve it?  but I will find proper time to merge it